apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -115,4 +127,19 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     return this._apiSetUrlDefault; // match not found.
   }
 
-}
+  private _fixBoldAndItalics(text: string): string {
+    while (text.indexOf('\\*') >= 0) {
```

Comment:
>while (text.indexOf('\\*') >= 0) { [](start = 4, length = 34)

This is an O(n^2) algorithm.

I would suggest to use [Text.replaceAll](https://microsoft.github.io/web-build-tools/api/node-core-library.text.replaceall.html) instead.

Reply:
Sweet! Good to know that exists.